README.md

```diff
@@ -51,7 +60,7 @@ Notes regarding Python 3.9 and Conda:
 ### Alternative: using Anaconda
 
 1. Open the Anaconda prompt and change to the directory where you checked out this repository.
-1. Ensure you have a working Anaconda Python 3.6 or 3.8 64bit installation (or additionally Python 3.7 on Windows). For other Python versions, please [build](https://github.com/Esri/pyprt) PyPRT yourself at the moment.
+1. Ensure you have a working Anaconda Python 3.8 or 3.9 64bit installation (or additionally Python 3.7 on Windows). For other Python versions, please [build](https://github.com/Esri/pyprt) PyPRT yourself at the moment.
```

Comment:
We're missing the 3.10 64bit installation mentioning here.

Reply:
True, please add it.